SocialCareCaseViewerApi.Tests/V1/Boundary/Request/FinishCaseSubmissionRequestTests.cs

```diff
@@ -11,12 +11,12 @@ public class FinishCaseSubmissionRequestTests
         [Test]
```

Comment:
We got to update the Class and file name

Reply:
Also the function names and variable names in here that have finish, think I left this by mistake, sorry